src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -797,6 +797,17 @@ function fs.complete(sPath, sLocation, bIncludeFiles, bIncludeDirs)
     return tEmpty
 end
 
+function fs.isMountPoint(sPath)
+    expect(1, sPath, "string")
+    if not fs.isDir(sPath) then
```

Comment:
_Technically_ files can also be mounts (see #90), so I'm not sure if we want this.